lib/engine/game/g_1822_africa/game.rb

```diff
@@ -642,7 +642,7 @@ def gold_mine_corp
         end
 
         def gold_mine_bonus(_route, stops)
-          return 0 unless stops&.find { |s| s.hex == @gold_mine_token.hex }
+          return 0 if !@gold_mine_token || !stops&.find { |s| s.hex == @gold_mine_token.hex }
```

Comment:
```suggestion
          return 0 if !@gold_mine_token || stops&.none? { |s| s.hex == @gold_mine_token.hex }
```
You're not using the return value from `find`, apart from converting it into a boolean. Calling `none?` instead is more idiomatic for Ruby.

Reply:
@ollybh thank you for the suggestion, I've changed the code. I'm still not used to Ruby having methods for all possible use-cases :)